pom.xml

```diff
@@ -219,6 +219,7 @@
                     </execution>
                 </executions>
             </plugin>
+            <!--
```

Comment:
Please, remove the commented plugins if you're not planning to use them

Reply:
Good find, removed them.